docs/developer/explanation/code.rst

```diff
@@ -3,56 +3,32 @@ Launchpad Code Hosting
 
 .. include:: ../../includes/important_not_revised.rst
 
-Launchpad hosts source code using `Git <https://git-scm.com/>`__ and
-`Bazaar <https://www.breezy-vcs.org/>`__ branches. However dealing
-with code in Launchpad covers many more areas than just hosting
+Launchpad hosts source code using `Git <https://git-scm.com/>`__ branches. 
+However dealing with code in Launchpad covers many more areas than just hosting
 branches.
 
 Code for the web application is only part of the Launchpad codehosting
 system. The major sub-systems are:
 
--  The `git` and `bzr` / `brz` clients (neither of which is part of
-   Launchpad, but their behaviours are important to us)
--  Connectivity to Launchpad (git, git+ssh, and https for Git; SFTP and
-   bzr+ssh for Bazaar)
+-  The `git` client (which is not a part of Launchpad but its behaviour is 
+   important to us).
+-  Connectivity to Launchpad (git, git+ssh, and https for Git)
 -  Hosting infrastructure
 -  The underlying object model
 -  The web application
 -  Email processing
--  Code imports (from CVS, Subversion, git and Mercurial)
+-  Code imports from git 
```

Comment:
Afaik we did not remove the functionality to import from other code versioning systems, we simply removed the possibility to import into bazaar